lib/grizzly/zwave/commands/switch_multilevel_set.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Grizzly.ZWave.Commands.SwitchMultilevelSet do
+  @moduledoc """
+  What does this command do??
```

Comment:
Can we update this to say something useful?

Sorry for picky on this, last time with Grizzly before the release I punted on docs went through each command and had to update them and it was very tedious.

It does not have to be anything too long, but I'd just prefer we had something.

Reply:
No problem. I had a feeling I had neglected one.

Reply:
Done